Units/parser-inko.r/everything.d/expected.tags

```diff
@@ -0,0 +1,18 @@
+A	input.inko	/^object A$/;"	o
+in_a	input.inko	/^  def in_a {$/;"	m	object:A
+another	input.inko	/^  def another {}$/;"	m	object:A
+foo	input.inko	/^def foo {}$/;"	m
+B	input.inko	/^object B {$/;"	o
+in_b	input.inko	/^  def in_b {$/;"	m	object:B
+C	input.inko	/^  object C {$/;"	o	object:B
+nested_method	input.inko	/^    def nested_method {}$/;"	m	object:B.C
+global	input.inko	/^def global {}$/;"	m
+SomeTrait	input.inko	/^trait SomeTrait {$/;"	t
+Socket	input.inko	/^object Socket impl Foo {$/;"	o
+quack	input.inko	/^  def quack {}$/;"	m	object:Socket
+Socket	input.inko	/^impl Bar for Socket {$/;"	r	implements:Bar
+moo	input.inko	/^  def moo !! Integer {}$/;"	m	reopen:Socket
+Chickens	input.inko	/^impl Chickens {$/;"	r
+bok_bok	input.inko	/^  def bok_bok {}$/;"	m	reopen:Chickens
+NUMBER	input.inko	/^let NUMBER = 10$/;"	c
+@number	input.inko	/^let @number = 20$/;"	a
```

Comment:
Do you think `@` is the part of the attribute name?
(I'm not sure.)

Reply:
Yes, the language treats them as being part of the name (`@foo` and `foo` are thus different attributes).